config/get_hypre_version.cpp

```diff
@@ -18,12 +18,16 @@
 #define HYPRE_VERSION_STRING HYPRE_PACKAGE_VERSION
 #endif
 
+// Macros to expand a macro as a string
+#define STR_EXPAND(s) #s
+#define STR(s) STR_EXPAND(s)
```

Comment:
OK, Google is right (at least this time). :-) I guess the explanation is that we are using `STR()` with another macro, so it has to be expanded twice.